coffee-module/coffee-module-mp/coffee-module-mp-restclient/src/main/java/hu/icellmobilsoft/coffee/module/mp/restclient/provider/DefaultLoggerClientResponseFilter.java

```diff
@@ -139,21 +139,14 @@ protected String logEntity(ClientRequestContext requestContext, ClientResponseCo
             return null;
         }
         InputStream in = responseContext.getEntityStream();
-        if (in == null) {
-            return "";
-        }
-        ByteArrayOutputStream out = new ByteArrayOutputStream();
         try {
-            IOUtils.copy(in, out);
-
-            byte[] responseEntity = out.toByteArray();
-            int maxResponseEntityLogSize = RestLoggerUtil.getMaxEntityLogSize(requestContext, LogSpecifierTarget.CLIENT_RESPONSE);
-            if (maxResponseEntityLogSize != LogSpecifier.NO_LOG &&
-            // byte-code betoltesi fajlokat ne loggoljuk ki egeszben
-                    Objects.equals(responseContext.getMediaType(), MediaType.APPLICATION_OCTET_STREAM_TYPE)) {
-                maxResponseEntityLogSize = RequestResponseLogger.BYTECODE_MAX_LOG;
+            int maxResponseEntityLogSize = getMaxResponseEntityLogSize(requestContext, responseContext);
 
+            if (in == null) {
+                return requestResponseLogger.printEntity(null, maxResponseEntityLogSize, RequestResponseLogger.RESPONSE_PREFIX);
```

Comment:
ezzel szinten null lessz csak loggolva

Reply:
Nem mert kicsit módosítottam a `printEntity` metódust is, hogy rendesen kezelje a null-t is.

Reply:
oks, a migracios doksiba valamit irjunk hogy listazzuk a kod valtozast...